js/bybit.js

```diff
@@ -3114,8 +4091,12 @@ module.exports = class bybit extends Exchange {
         const market = this.market (symbol);
         symbol = market['symbol'];
         const isUsdcSettled = (market['settle'] === 'USDC');
+        let enableUnifiedMargin = undefined;
+        [ enableUnifiedMargin, params ] = this.handleOptionAndParamsValue (params, 'createOrder', 'enableUnifiedMargin', false);
```

Comment:
@sc0Vu aren't we missing the v3 upgrade here? https://bybit-exchange.github.io/docs/derivativesV3/contract/#t-dv_placeorder

Reply:
There is a `createUnifiedMarginOrder` to submit v3 order. Is anything wrong in that function?

Reply:
@carlosmiei it seems we miss some contract v3 endpoint, I'll add.